runtime/js/90_deno_ns.js

```diff
@@ -178,6 +178,11 @@ const denoNsUnstable = {
   Kv: kv.Kv,
   KvU64: kv.KvU64,
   KvListIterator: kv.KvListIterator,
+  jupyter: {
```

Comment:
I think `Deno.jupyter` should only exist if we're running within the context of `deno jupyter`

Reply:
I actually want to make it a getter that will throw an informative error in non-Jupyter context 👍